app/views/articles/_actions.html.erb

```diff
@@ -40,9 +40,9 @@
             class="flex justify-between crayons-link crayons-link--block w-100 bg-transparent border-0"
```

Comment:
could you please unbundle the translation changes from this PR? They are unrelated, thank you!

Reply:
Hey, @rhymes could you please help I pushed the changes but I don't know how to remove the translations changes.
Query:
In my local, I did try to move my git HEAD to the point where I cloned the repo and kept the prop types fixes and committed but haven't pushed. Is this okay?